scripts/jenkins/Jenkinsfile

```diff
@@ -587,7 +587,7 @@ def generateFullBuildStages() {
     CMAKE_FLAGS_BUILD_ALL +
       CMAKE_FLAGS_DEBUG +
       CMAKE_FLAGS_CLANG + [
-      'PLUGINS': 'ALL;-crypto;-fcrypt',
+      'PLUGINS': 'ALL;-crypto;-fcrypt;-gpgme',
```

Comment:
I see a different diff for `Jenkinsfile` locally? :confused: Do you need to rebase or is this a github problem?

Reply:
Actually, the question is: can these tests be enabled again for `debian-unstable-full-clang` or do they still cause problems?

Reply:
Ah you re-enabled the unit tests :-) Should we leave them enabled or not?

Reply:
Let's try them, but I think they still caused problems the last time.

Reply:
Thank you for reacting so quickly. You can also leave some tests disabled, but please open another PR where they are enabled (or an issue?), so we do not forget that it needs to be fixed.